install.sh

```diff
@@ -0,0 +1,14 @@
+#! /usr/bin/env bash
+
+if [[ $ASSETS == 'nobuild' ]]; then
+    composer install --no-interaction --no-progress --prefer-dist -o
+else
+    echo "Installing PHP dependencies through Composer..."
+    composer install
```

Comment:
I thought the user may prefer having the output of this command.

Yup, the user will just have to run `./install.sh`. I can add the `wallabag:install` command to the script too.